gcn/notices/maxi/gsc.unknown.example.json

```diff
@@ -0,0 +1,29 @@
+{
+  "$schema": "https://gcn.nasa.gov/schema/main/gcn/notices/maxi/gsc.unknown.schema.json",
+  "instrument": "GSC",
+  "trigger_time": "2023-02-20T08:56:01.00Z",
+  "id": ["9995289617"],
+  "ra": 346.56,
+  "dec": 18.61,
+  "circle_error_radius": 1.0,
+  "elliptical_error_radii": [0.22, 0.19],
+  "elliptical_error_roll": 153.0,
+  "rectangular_error_corners": [
+    [345.72, 19.547],
+    [345.464, 19.29],
+    [347.309, 17.634],
+    [347.566, 17.889]
```

Comment:
Would you produce one of circle, ellipse, or rectangle for each source detection?  I don't understand how you would produce all 3.  If it's one of those, then 3 examples would be needed.  You could also consider embedding HEALPix maps.